src/coreclr/vm/appdomain.cpp

```diff
@@ -3096,10 +3096,10 @@ DomainAssembly *AppDomain::LoadDomainAssemblyInternal(AssemblySpec* pIdentity,
     {
         AssemblySpec spec;
         spec.InitializeSpec(result->GetFile());
-        if (spec.CanUseWithBindingCache() && result->CanUseWithBindingCache())
+        if (result->CanUseWithBindingCache())
```

Comment:
Do these other `CanUseWithBindingCache` methods always return true  as well?

Reply:
Indeed, they do.
`PEAssembly::CanUseWithBindingCache` - and thus `DomainAssembly::CanUseWithBindingCache` - always returns true. (`PEFile::CanUseWithBindingCache` does not appear to be used.)